bin/contact-search-service

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env python
 """ Start the contact search indexing service. """
 import os
-from setproctitle import setproctitle
 
 import click
 import gevent_openssl
+from setproctitle import setproctitle
+
 gevent_openssl.monkey_patch()
```

Comment:
While this script is probably not used at all, ideally the only import above this line would be `gevent_openssl`.